packages/mui-base/src/PreviewCard/Arrow/PreviewCardArrow.tsx

```diff
@@ -56,6 +57,21 @@ const PreviewCardArrow = React.forwardRef(function PreviewCardArrow(
   return renderElement();
 });
 
+namespace PreviewCardArrow {
+  export interface OwnerState {
+    open: boolean;
+    side: Side;
+    alignment: Alignment;
+  }
```

Comment:
nit: leave a blank line between definitions
```suggestion
  }
  
```